File: jvm/src/test/kotlin/advent2018Kotlin/test/D02Tests.kt
package advent2018Kotlin.test

import advent2018.Utils
import advent2018Kotlin.D02
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test
import java.nio.file.Files

class D02Tests {
    @Test
    fun input2() {
        assertThat(D02.input2(Files.readAllLines(Utils.of("d02.txt"))))
                .isEqualTo("mxhwoglxgeauywfkztndcvjqr");
    }
}

File: dotnet/D02Tests.cs
using Xunit;
using FluentAssertions;
using System.IO;

namespace Advent2018
{
    public class D2Tests
    {
        [Fact]
        public void Input2()
        {
            var lines = File.ReadAllLines(FileLocator.Input("d02.txt"));
            D2.FindClosestLines(lines).Should().Be("mxhwoglxgeauywfkztndcvjqr");
        }
    }
}
